src/components/StatusChip.tsx

```diff
@@ -0,0 +1,79 @@
+import BlockIcon from '@mui/icons-material/Block';
+import CheckIcon from '@mui/icons-material/Check';
+import HourglassEmptyIcon from '@mui/icons-material/HourglassEmpty';
+import { Box, Typography } from '@mui/material';
+import { StandardCSSProperties } from '@mui/system';
+import { useTranslation } from 'react-i18next';
+import { NviCandidateStatus } from '../types/nvi.types';
+import { Ticket } from '../types/publication_types/ticket.types';
+
+const ticketColor = {
+  PublishingRequest: 'publishingRequest.main',
+  DoiRequest: 'doiRequest.main',
+  GeneralSupportCase: 'generalSupportCase.main',
+};
+
+interface TicketStatusChipProps {
+  ticket: Ticket;
+}
+
+export const TicketStatusChip = ({ ticket }: TicketStatusChipProps) => {
+  const { t } = useTranslation();
+
+  if (ticket.status === 'NotApplicable' || ticket.type === 'UnpublishRequest') {
+    return null;
+  }
+
+  const text = t(`my_page.messages.ticket_types.${ticket.status}`);
+
+  if (ticket.status === 'Completed') {
+    return <StatusChip text={text} icon="check" color={ticketColor[ticket.type]} />;
+  }
+
+  return <StatusChip text={text} icon={ticket.status === 'Closed' ? 'block' : 'hourglass'} color="secondary.dark" />;
+};
+
+interface NviStatusChip {
+  status: NviCandidateStatus;
+}
+
+export const NviStatusChip = ({ status }: NviStatusChip) => {
+  const { t } = useTranslation();
+
+  const text = t(`tasks.nvi.status.${status}`);
+
+  if (status === 'Approved') {
+    return <StatusChip text={text} icon="check" color="nvi.main" />;
+  }
+
+  return <StatusChip text={text} icon={status === 'Rejected' ? 'block' : 'hourglass'} color="secondary.dark" />;
+};
+
+interface StatusChipProps {
+  text: string;
+  icon: 'check' | 'block' | 'hourglass';
+  color?: StandardCSSProperties['color'];
+}
+
+export const StatusChip = ({ text, color, icon }: StatusChipProps) => {
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        gap: '0.2rem',
+        alignItems: 'center',
+        bgcolor: color ?? 'secondary.dark',
+        p: '0.25rem 0.5rem',
```

Comment:
```suggestion
        py: '0.25rem',
        pl: '0.5rem',
        pr: icon ? '0.75rem' : '0.5rem',
```
For å få lik spacing på begge sider av Chip'en

Reply:
Ser at paddingen egentlig er riktig, det er bare at ikonet er litt mindre enn teksten (rød), og også har litt tomrom på sidene (blå). Så må nok går for noe sånt ja, tenker jeg satser på `p: '0.25rem 0.75rem 0.25rem 0.5rem'`
![bilde](https://github.com/user-attachments/assets/9d7b93f6-e91b-43ef-a694-d84c0c15e74f)